examples/dos-games/zarf.yaml

```diff
@@ -4,6 +4,11 @@ metadata:
   description: Simple example to load classic DOS games into K8s in the airgap
   version: 1.2.0
 
+# The documentation key can be utilized to pass information to package consumers
+# Commonly used to detail package configuration, or as in this instance, application specific documentation.
+documentation:
+  walkthrough: "secrets.md"
```

Comment:
not a blocker - but there is a drift here between existing consistency for how we specify files and this implementation. The use of a map[string]string is captured in the PR description but nowhere else as far as I can tell. 

I believe we could use more supporting documentation here - but the other note was just validating that we want to continue with this approach (I believe we do). 

said another way - we typically use `files` as a list and this approach deviates from that - I believe we want documentation to describe the current format and expectations?